gemfiles/elasticsearch_ruby.gemfile.lock

```diff
@@ -0,0 +1,50 @@
+PATH
```

Comment:
Lock files should not be here. Probably it's because this change was done before the cleaning commit.